explorer/ExplorerProgram.ts

```diff
@@ -48,6 +51,7 @@ interface ExplorerGrapherInterface extends GrapherInterface {
     yScaleToggle?: boolean
     yAxisMin?: number
     facetYDomain?: FacetAxisDomain
+    selectedFacetStrategy?: FacetStrategy
```

Comment:
There is already `facet` on `GrapherInterface`, should that be removed?

Reply:
Turns out this property wasn't needed on `ExplorerGrapherInterface`, removed it.